app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -8,6 +8,22 @@ def create
     end
   end
 
+  def destroy
+    # fetch refresh token from request header
+    refresh_token = request.headers["Authorization"]&.split(" ")&.last
```

Comment:
Please keep tokens handling just in the JSON response for consistency with the reset of the code. Do not phrase the header.